android/src/main/java/com/reactnative/ivpusic/imagepicker/PickerModule.java

```diff
@@ -305,6 +307,10 @@ private void initiateCamera(Activity activity) {
 
             cameraIntent.putExtra(MediaStore.EXTRA_OUTPUT, mCameraCaptureURI);
 
+            if (this.useFrontCamera) {
+                cameraIntent.putExtra("android.intent.extras.CAMERA_FACING", 1);
```

Comment:
You also need to add these two lines, this flag doesn't work on the Google Pixel 
cameraIntent.putExtra("android.intent.extras.LENS_FACING_FRONT", 1);
cameraIntent.putExtra("android.intent.extra.USE_FRONT_CAMERA", true);

Reply:
@juanjonucleus could you send a PR with this fix?